keras/src/export/export_utils.py

```diff
@@ -105,3 +147,38 @@ def convert_spec_to_tensor(spec, replace_none_number=None):
             s if s is not None else replace_none_number for s in shape
         )
     return ops.ones(shape, spec.dtype)
+
+
+# Registry for export formats
```

Comment:
Let's clean up the code below here in this file. I don't think we need any of it. Instead add we can just directly handle calling a lite rt export function in `model.export()` in mode.py.